packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -255,6 +255,6 @@ export type IncomingPaymentJSON = {
   externalRef?: string
   createdAt: string
   updatedAt: string
-  expiresAt: string
+  expiresAt?: string
```

Comment:
I thought we had that as required since it is either set to the client's value or the server's default.

Reply:
That's correct for Rafiki the Open Payments server.
But Rafiki the client of an arbitrary Open Payments server has to go by the OpenAPI spec in which `expiresAt` isn't required:
https://github.com/interledger/open-payments/blob/6d128e2298301a3a76d2421581e2ffa097ebf33c/open-api-spec.yaml#L915-L939